ci/Vagrantfile

```diff
@@ -17,7 +17,7 @@ Vagrant.configure("2") do |config|
   config.nfs.functional = false
 
   config.vm.synced_folder "../.", "/opt/vmck", type: "rsync",
-      rsync__exclude: [".git/", ".pytest_cache/", "data/", "docs/"]
+      rsync__exclude: ["--ignore-from", "../.gitignore"]
```

Comment:
Is this doing what you think it's doing? I believe this ignores files named `--ignore-from` and `./.gitignore`, not what we want. See this: https://www.vagrantup.com/docs/synced-folders/rsync.html#rsync__exclude

The dance here seems to be:
- add `--ignore-from` `./.gitignore` to the default `rsync_args` you see in the docs above
- do nothing to the `rsync_exclude` arg